src/core/infrastructure/shortcut.js

```diff
@@ -0,0 +1,57 @@
+// import EventListener from './event.listener'; - TODO:: use instead of document
+
+export default function Shortcut(document) {
```

Comment:
Can you please briefly describe shortcut contract in docs? I mean, describe delimiters ( | +) and a list of encoded values